### PR TITLE
Move to babel 6 (closes #163)

### LINKED ISSUE #163: Move to babel 6

### PR DESCRIPTION
**Pull request by dfilatov**

### CODE REVIEW

File: spec/patch/patch.spec.js [hunk 1]
```diff
@@ -9,49 +9,49 @@ describe('patch', () => {
...
+        require('./data/complex-insert-to-ending-without-key').default,
+        require('./data/complex-remove-from-beginning-with-key').default,
+        require('./data/complex-remove-from-beginning-without-key').default,
+        require('./data/complex-remove-from-ending-with-key').default,
+        require('./data/complex-remove-from-ending-without-key').default,
+        require('./data/complex-reverse').default,
+        require('./data/complex-shuffle-with-inserts-removes').default
     ];
 
     data.forEach(specData => {
```
Comment (zxqfox) [inline spec/patch/patch.spec.js]: If you gonna use `require` you can add `{specData: default}` here.

Comment (dfilatov) [inline spec/patch/patch.spec.js]: It doesn't work (

Comment (dfilatov) [inline spec/patch/patch.spec.js]: @zxqfox It doesn't work (

Comment (zxqfox) [inline spec/patch/patch.spec.js]: My bad, it should be: `{default: specData}`.

[example on babel/repl](http://babeljs.io/repl/#?experimental=false&evaluate=true&loose=false&spec=false&code=var%20data%20%3D%20%5B%20require('qwe')%20%5D%3B%0A%0Adata.forEach((%7Bdefault%3A%20specData%7D)%20%3D%3E%20%7B%0A%20%20%2F%2F%20yo%20%0A%7D)%3B%0A)

Comment (dfilatov) [inline spec/patch/patch.spec.js]: @zxqfox Thanks, fixed

**Pull request merged**